pkg/config/pipeline.go

```diff
@@ -332,11 +332,17 @@ func (p *PipelineConfig) Update(request *rpc.StartEgressRequest) error {
 
 		p.Info.RoomName = req.TrackComposite.RoomName
 		if audioTrackID := req.TrackComposite.AudioTrackId; audioTrackID != "" {
+			if strings.HasPrefix(audioTrackID, "TR_V") {
+				return errors.ErrInvalidInput("audio_track_id")
+			}
 			p.AudioEnabled = true
 			p.AudioTrackID = audioTrackID
 			p.AudioTranscoding = true
 		}
 		if videoTrackID := req.TrackComposite.VideoTrackId; videoTrackID != "" {
+			if strings.HasPrefix(videoTrackID, "TR_A") {
+				return errors.ErrInvalidInput("video_track_id")
+			}
```

Comment:
If track source is not set, I think the A/V prefix will not be there IIRC.

Reply:
I think that's fine provided we never have a valid video track with a "A" prefix? We may not catch every error case though indeed.

Reply:
I had no idea you could have a track ID without a prefix. I'll have to check it at another point then